obj/objects.h

```diff
@@ -68,9 +68,9 @@ namespace objects
     class EnergyCollector : public Sphere
     {
     public:
-        EnergyCollector(const core::Vec3 &origin, double energy) : _id(id), _energy(energy)
+        EnergyCollector(const core::Vec3 &origin, double energy = 0) : _energy(energy)
```

Comment:
Why is the default value useful here? In which circumstances you want to use a non-default value?

Reply:
you are right, its not important ;-)